README.md

```diff
@@ -1,96 +1,31 @@
+[![MELPA](https://melpa.org/packages/rust-mode-badge.svg)](https://melpa.org/#/rust-mode)
+
 `rust-mode`: A major Emacs mode for editing Rust source code
 ============================================================
 
-`rust-mode` makes editing [Rust](http://rust-lang.org) code with Emacs
-enjoyable.  `rust-mode` requires Emacs 24 or later.
```

Comment:
I would keep emacs 24 as the minimally supported version

Reply:
Somebody mentioned there was a pull request lately that broke emacs24 compatibility.

Reply:
@brotzeit this has been resolved and the code is still emacs24 compatible. Version 24 is still tested in CI.